inbm/build.sh

```diff
@@ -4,14 +4,40 @@ set -euxo pipefail
 DOCKER_CONTENT_TRUST=0
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
+# arg supported: --build-windows=true or --build-windows=false
+# Note, "--build-windows true / --build-windows false" will not work
+# Default is true
+# second arg: build-check; default true. false skips unit tests/mypy/other checks
+
+# Default arguments
+build_windows=true
+build_check=true
+
+# Parse command line arguments
+for arg in "$@"; do
```

Comment:
<h3>$\color{#FFA500}Improvements: Code Maintainability$</h3>

Add a check to ensure that the arguments `--build-windows` and `--build-check` are explicitly set to either `true` or `false` to avoid ambiguity.